DQM/SiStripMonitorClient/bin/BuildFile.xml

```diff
@@ -6,3 +6,4 @@
 <bin   name="ls_cert" file="ls_cert.cc"></bin>
 <bin   name="lsbs_cert" file="lsbs_cert.cc"></bin>
 <bin   name="moduleOccupancyPlots" file="moduleOccupancyPlots.cc"></bin>
+<bin   name="moduleOccupancyTrend" file="moduleOccupancyTrend.cc"></bin>
```

Comment:
Perhaps you should include 'tracker' into the name?

In any case, you are polluting the global 'executable' path, but I do not know of any better way to put this executable. But then this is not really a test...
Perhaps you can try to put it in "test/" subfolder.